testcases/two_zone_apartment_hydronic/models/TwoZoneApartmentHydronic/Components/Thermostat_T.mo

```diff
@@ -0,0 +1,106 @@
+  parameter Modelica.SIunits.Temperature TSetHeaOcc = 273.15+21 "Occupied heating setpoint Monday to Friday" annotation (Dialog(group="Setpoints"));
+  parameter Modelica.SIunits.Temperature TSetHeaWee = 273.15+16 "Heating setpoint Saturday and Sunday" annotation (Dialog(group="Setpoints"));
+  parameter Real bandwidth=2 "Bandwidth around reference signal";
+  Modelica.Blocks.Interfaces.RealInput TZon "Measured zone air temperature"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}})));
+  Modelica.Blocks.Sources.CombiTimeTable TSetHea(
+    smoothness=Modelica.Blocks.Types.Smoothness.ConstantSegments,
+    extrapolation=Modelica.Blocks.Types.Extrapolation.Periodic,
+    table=[0,TSetHeaUno; occSta,TSetHeaOcc; occEnd,TSetHeaUno; 86400,TSetHeaUno;
+        occSta + 86400,TSetHeaOcc; occEnd + 86400,TSetHeaUno; 86400*2,
+        TSetHeaUno; occSta + 86400*2,TSetHeaOcc; occEnd + 86400*2,TSetHeaUno;
+        86400*3,TSetHeaUno; occSta + 86400*3,TSetHeaOcc; occEnd + 86400*3,
+        TSetHeaUno; 86400*4,TSetHeaUno; occSta + 86400*4,TSetHeaOcc; occEnd +
+        86400*4,TSetHeaUno; 86400*5,TSetHeaUno; occSta + 86400*5,TSetHeaWee;
+        occEnd + 86400*5,TSetHeaWee; 86400*6,TSetHeaWee; occSta + 86400*6,
+        TSetHeaWee; occEnd + 86400*6,TSetHeaWee; 86400*7,TSetHeaWee])
+                 "Heating temperature setpoint for zone air"
+    annotation (Placement(transformation(extent={{-100,-70},{-80,-50}})));
+  Modelica.Blocks.Interfaces.RealOutput ValCon "Zone valve control"
+    annotation (Placement(transformation(extent={{100,50},{120,70}})));
+  Modelica.Blocks.Interfaces.RealOutput TSetZ "Supply temperature setpoint"
+    annotation (Placement(transformation(extent={{100,-70},{120,-50}})));
+  Modelica.Blocks.Logical.OnOffController onOffCon(bandwidth=bandwidth)
+    annotation (Placement(transformation(extent={{-60,50},{-40,70}})));
+  Modelica.Blocks.Math.BooleanToReal booleanToReal
+    annotation (Placement(transformation(extent={{-12,50},{8,70}})));
+  Modelica.Blocks.Logical.GreaterEqualThreshold greaterEqualThreshold(threshold=
+       TSetHeaUno)
+    annotation (Placement(transformation(extent={{-14,-10},{6,10}})));
+  Modelica.Blocks.Interfaces.RealOutput Occ
+    "If value is one thermal zone is occupied else it is empty"
+    annotation (Placement(transformation(extent={{100,-10},{120,10}})));
+  Modelica.Blocks.Math.BooleanToReal booleanToReal1
+    annotation (Placement(transformation(extent={{40,-10},{60,10}})));
+  IBPSA.Utilities.IO.SignalExchange.Overwrite oveTsetZon(u(
+      unit="K",
+      min=273.15,
+      max=273.15 + 45), description="Setpoint temperature for thermal zone")
+    "Overwrite Setpoint temperature for thermal zone"
+    annotation (Placement(transformation(extent={{-66,-70},{-46,-50}})));
+  IBPSA.Utilities.IO.SignalExchange.Read reaTsetZon(y(unit="K"), description=
```

Comment:
Since [v0.2.0 ](https://github.com/ibpsa/project1-boptest/blob/v0.2.0/releasenotes.md)and closing of #364, overwrite blocks don't need separate read blocks to create outputs.  Users can request data of overwrite point and get value used, either embedded control or overwritten.  I suggest removing to stay consistent and also to reduce the number of output points that are not needed.

Reply:
Removed all redundant read blocks and updated documentation accordingly